### PR TITLE
fix: update dependency typings

### PR DESCRIPTION
**Pull request by wkillerud**
Update dependencies and internal typings

### CODE REVIEW

File: lib/layout.js [hunk 1]
```diff
@@ -294,14 +290,12 @@ export default class PodiumLayout {
 
         // Join metric streams
         this.httpProxy.metrics.pipe(this.metrics);
-        // @ts-ignore
         this.context.metrics.pipe(this.metrics);
         this.client.metrics.pipe(this.metrics);
 
         // Register proxy endpoints
-        // @ts-ignore
+        // @ts-expect-error Not typed yet. Coming in a future TypeScript Tuesday™
```
Comment (wkillerud) [inline lib/layout.js:297]: TODO: `@podium/client` types from JSDoc

File: lib/layout.js [hunk 2]
```diff
@@ -561,8 +555,9 @@ export default class PodiumLayout {
         incoming.css = [...this.cssRoute];
         incoming.js = [...this.jsRoute];
 
-        // @ts-ignore
+        // @ts-expect-error Context hasn't been updated yet with the new typings for HttpIncoming.
```
Comment (wkillerud) [inline lib/layout.js:558]: This should be handled by renovate-bot after a while

File: lib/layout.js [hunk 3]
```diff
@@ -561,8 +555,9 @@ export default class PodiumLayout {
...
         if (context) await this.context.process(incoming);
+        // @ts-expect-error Proxy hasn't been updated yet with the new typings for HttpIncoming.
```
Comment (wkillerud) [inline lib/layout.js:560]: TODO: `@podium/proxy` types from JSDoc, although this one in particular might be handled by renovate-bot

**Review by digitalsadhu — APPROVED** [review]
Lovely. Appreciate the inline PR comments for context... err... no pun intended

File: lib/layout.js (hunk 1 above)
Comment (wkillerud) [reply lib/layout.js:297]: Client types PR https://github.com/podium-lib/client/pull/377

**Pull request merged**